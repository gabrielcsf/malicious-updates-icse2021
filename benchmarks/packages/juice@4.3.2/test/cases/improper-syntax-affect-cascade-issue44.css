body {
  padding: 5px;
}
a {
  color: ;
}
table {
  padding: 5px;
  color: #333;
}