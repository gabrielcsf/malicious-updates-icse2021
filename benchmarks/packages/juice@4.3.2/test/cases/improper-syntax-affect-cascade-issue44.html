<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Testing</title>
</head>
<body>
  <table>
    <tr>
      <td><a href="sdf">a test!</a></td>
    </tr>
  </table>
</body>
</html>