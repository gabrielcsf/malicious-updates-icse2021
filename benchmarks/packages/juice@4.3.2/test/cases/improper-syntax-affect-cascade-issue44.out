<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Testing</title>
</head>
<body style="padding: 5px;">
  <table style="padding: 5px; color: #333;">
    <tr>
      <td><a href="sdf">a test!</a></td>
    </tr>
  </table>
</body>
</html>